bpf/proxy/lb_src_range_test.go

```diff
@@ -57,7 +57,7 @@ var _ = Describe("BPF Load Balancer source range", func() {
 				net.IPv4(10, 0, 0, 2),
 				2222,
 				v1.ProtocolTCP,
-				proxy.K8sSvcWithExternalIPs([]string{"35.0.0.2"}),
+				proxy.K8sSvcWithLoadBalancerIPs([]string{"35.0.0.2"}),
```

Comment:
I wonder if we should execute this test 2x, once for ExternalIPs and once for LBIPs

Reply:
yes. Will add one more test for external IP

Reply:
We may just turn this set of tests into a function and run it 2x with 2 differnt options :thinking: